### PR TITLE
Update rsync command

### PR DESCRIPTION
**Pull request by thiagoalcav**
Adds --no-perms --no-owner params to rsync command for maintain compatibility with NFS volumes, else rsync change owner and perms of files synced causing error.

### CODE REVIEW

**Comment by monotek** [comment]
Could you please change the pr to make it an optional rsync config parameter?

**Comment by monotek** [comment]
Maybe somebody else want to use other options 
so a generic variable would be nice.

**Comment by thiagoalcav** [comment]
Hi! Please, check if this resolve the request. Note that i needed to add a specific comment to ignore spellcheck stage for double quotes as described in: https://github.com/koalaman/shellcheck/wiki/Ignore

...maybe only need one ignore instruction. feel free to correct or request another correction.

File: containers/zammad/docker-entrypoint.sh [hunk 1]
```diff
@@ -19,7 +19,9 @@ set -e
 : "${ZAMMAD_RAILSSERVER_PORT:=3000}"
 : "${ZAMMAD_WEBSOCKET_HOST:=zammad-websocket}"
 : "${ZAMMAD_WEBSOCKET_PORT:=6042}"
+: "${ZAMMAD_INSTALL_RSYNC_PARAMS:=--no-perms --no-owner}"
```
Comment (monotek) [inline containers/zammad/docker-entrypoint.sh]: i don't see this used anywhere?

Comment (thiagoalcav) [reply containers/zammad/docker-entrypoint.sh]: Please ignore that! All before this commit (https://github.com/zammad/zammad-docker-compose/pull/150/commits/f839feccb4eaef06c90778757ae5c4d5913b5e80), was testings to solve broken CI pipeline.

Comment (thiagoalcav) [reply containers/zammad/docker-entrypoint.sh]: The commit quoted above resolve the first request (export rsync params to a generic var). So all other PRs can be rejected if you prefer.

**Pull request merged**